src/averageAges.js

```diff
@@ -1,5 +1,16 @@
 'use strict';
 
+const INITIAL_VALUE = 0;
+const FEMALE_SEX = 'f';
+const MALE_SEX = 'm';
+const CENTURY_CHANGE = 100;
+
+const getAverageAge = people => {
+  const ageAmount = people.reduce((a, b) => a + b, INITIAL_VALUE);
+
+  return +(ageAmount / people.length).toFixed(2);
+};
```

Comment:
We take the exact value of each person's age at the entrance, so `people` is not quite suitable here.
```suggestion
const getAverageAge = ageValues => {
  const ageAmount = ageValues.reduce((a, b) => a + b, INITIAL_VALUE);

  return +(ageAmount / ageValues.length).toFixed(2);
};
```